sql/core/src/main/scala/org/apache/spark/sql/execution/datasources/FileScanRDD.scala

```diff
@@ -208,7 +199,6 @@ class FileScanRDD(
 
       override def close(): Unit = {
         updateBytesRead()
-        updateBytesReadWithFileSize()
```

Comment:
If we just remove this `updateBytesReadWithFileSize`, the issue in the description can be solved? We need to remove `updateBytesReadWithFileSize` in the line 142, too?

Reply:
Yes, before SPARK-19464,  there  will only one works between updateBytesRead and updateBytesReadWithFileSize.  If the hadoop version is 2.5 or  earlier, updateBytesReadWithFileSize works, If the hadoop version is 2.6 or later, updateBytesRead works.